app/javascript/packages/countdown/countdown-alert-element.spec.ts

```diff
@@ -41,4 +47,22 @@ describe('CountdownAlertElement', () => {
       expect(element.show).to.have.been.called();
     });
   });
+
+  it('redirects when time has expired', async () => {
+    const element = createElement({
+      redirectURL: '#teapot',
+      showAtRemaining: 60000,
+    });
+
+    sandbox.spy(element, 'handleRedirectTick');
+
+    sandbox.clock.tick(30000);
+    expect(element.handleRedirectTick).not.to.have.been.called();
+
+    sandbox.clock.tick(0);
+    expect(element.handleRedirectTick).to.have.been.called;
+
+    await waitFor(() => window.location.hash === '#teapot');
+    expect(window.location.hash).to.equal('');
```

Comment:
These two assertions seem at odds with one another. Also, I was surprised to find out that the `waitFor` here won't work as expected from my previous suggestion, since it appears to expect that an error would be thrown as long as it doesn't match:

```suggestion
    await waitFor(() => expect(window.location.hash).to.equal('#teapot'));
```

I think we're not waiting long enough in the lines above either, based on how the element creation is initialized at 1m32s from expiration. I think the full test looks something like this:

```ts
  it('redirects when time has expired', () => {
    createElement({ redirectURL: '#teapot' });

    sandbox.clock.tick(91000);
    expect(window.location.hash).to.equal('');
    sandbox.clock.tick(1000);
    expect(window.location.hash).to.equal('#teapot');
  });
```

Reply:
The one highlighted was a WIP--it didn't seem right, but I could not put my finger on the _why_.